relnotes/option.feature.md

```diff
@@ -0,0 +1,24 @@
+* `ocean.core.Option`
+
+  New module to add boolean flag indicating existence of a value to arbitrary
+  type in generic manner:
+
+  ```D
+      Option!(int) foo ( bool x )
+      {
+          if (x)
+              return option(42);
+          else
+              return undefined!(int);
+      }
+
+      foo(true).visit(
+          ()              { test(false); },
```

Comment:
Just for the sake of example simplicity: these `()` aren't necessary, are they?

Reply:
For empty arguments they are not, correct. I think in such situations it is better to still put them to simplify formatting and make "undefined" case more obvious.